store/sgxLevelDB/sgxLeveldb.go

```diff
@@ -0,0 +1,251 @@
+// Package sgxLevelDB implements a light client level db for panacea-doracle.
+// It does include Set & Get functions that are sealed & unsealed in the sgx environment.
+
+package sgxLevelDB
```

Comment:
pls rename this package to `sgxleveldb`.
https://go.dev/doc/effective_go#package-names

Reply:
Let's change the package name as @youngjoon-lee said. :)